packages/jsii-pacmak/lib/targets/dotnet/dotnetdocgenerator.ts

```diff
@@ -185,8 +185,8 @@ export class DotNetDocGenerator {
   }
 
   private prefixDisclaimer(translated: Translation) {
-    if (!translated.didCompile && INCOMPLETE_DISCLAIMER_NONCOMPILING) {
-      return `// ${INCOMPLETE_DISCLAIMER_NONCOMPILING}\n${translated.source}`;
+    if (translated.disclaimer) {
+      return `// ${translated.disclaimer}\n${translated.source}`;
```

Comment:
I don't think there's a big downside to just having the disclaimer be a part of `translated.source` here, do you?